docs/griptape-framework/drivers/src/prompt_drivers_structured_output.py

```diff
@@ -0,0 +1,35 @@
+import schema
+from rich.pretty import pprint
+
+from griptape.drivers import OpenAiChatPromptDriver
+from griptape.rules import JsonSchemaRule, Rule
+from griptape.structures import Pipeline
+from griptape.tasks import PromptTask
+
+pipeline = Pipeline(
+    tasks=[
+        PromptTask(
+            prompt_driver=OpenAiChatPromptDriver(
+                model="gpt-4o",
+                use_native_structured_output=True,
+                native_structured_output_strategy="native",
```

Comment:
Seems weird to see `use_native_...` then `native...strategy="native"`. Maybe just remove `native` from the param names?